pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +922,40 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAOperationsDuration tracks the duration of the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
```

Comment:
Should or should not parts of kubelet be treated as their own subsystem? `csi` is its own subsystem despite being part of kubelet (`csi_operations_duration`).

We now have `kubelet_dra_operations_duration_seconds`. If we were to use `dra` as subsystem, it would become shorter (`dra_operations_duration_seconds`).

@logicalhan: any thoughts on best practices here?

Reply:
@SergeyKanzhelev: do you perhaps have any preference here?

Reply:
I would think `dra_` should be enough. The only issue will be if we will start exposing `dra_` prefixed metrics from API server or some other component. I am not sure if we will be able to distinguish those based on some dimension like node name or something after metrics were collected. Maybe @logicalhan can help understand if there are any issues with dra as it's own subcomponent

Reply:
`csi` as sub-system is already doing the same. I'm pretty sure metric collection on a node from kubelet can be distinguished from metric collection of a control plane, because only node metrics will have the node name as additional label. They have to have that, otherwise it would be impossible to identify problematic nodes.

Reply:
my thinking exactly. I was just not sure if I am missing anything

Reply:
done